### PR TITLE
Integration Test for PostgreSQL and Manticore using Docker

### PR DESCRIPTION
**Pull request by PavelShilin89**
**Type of Change:** 
- Enhancement - Integration Test for PostgreSQL and Manticore using Docker

**Description of the Change:**
- Integration Test for PostgreSQL and Manticore using Docker

This test validates the correct interaction between PostgreSQL and Manticore using Docker containers. The following steps are performed during the test:

1. **Docker Daemon Initialization**: The Docker environment is initialized using the `dockerd` command, ensuring the Docker API is available for container management.
2. **PostgreSQL Container Launch**:
   - A PostgreSQL database is initialized using a Docker image.
   - The SQL script (`init.sql`) is loaded and executed, creating tables and inserting test data.
   - The readiness of PostgreSQL to accept connections is verified.
3. **Manticore Container Launch**:
   - The Manticore container is configured to index data from PostgreSQL using the provided configuration file.
   - The readiness of Manticore to accept connections is checked after startup.
4. **Indexing Verification**:
   - Manticore is queried via the MySQL protocol to ensure that the data from the PostgreSQL table has been correctly indexed and is available for search.

The test concludes with a successful verification that data from PostgreSQL is present in Manticore’s index.

**Related Issue (provide the link):**
https://github.com/manticoresoftware/docker/issues/88

### CODE REVIEW

**Comment by PavelShilin89** [comment]
@sanikolaev please verify implementation and approve for merger.

**Review by sanikolaev — CHANGES_REQUESTED** [review]
Do we really need the .rep files?

File: clt_tests/tests/test-postgres-docker.rec [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+––– input –––
+docker network create app-network --driver bridge > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+docker run -d --name postgres --network app-network -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=qazQASD! -e POSTGRES_DB=api_db -p 5432:5432 -v $(pwd)/clt_tests/tests/base/init.sql:/docker-entrypoint-initdb.d/init.sql postgres:latest > /dev/null 2>&1; echo $?
+––– output –––
+0
+––– input –––
+sleep 5; timeout 30 grep -m2 'database system is ready to accept connections' <(docker logs -f postgres 2>&1)
```
Comment (sanikolaev) [inline clt_tests/tests/test-postgres-docker.rec:11]: Why not just timeout 35? Is there any difference?

**Review by sanikolaev — CHANGES_REQUESTED** [review]

File: clt_tests/tests/test-postgres-docker.rec (hunk 1 above)
Comment (PavelShilin89) [reply clt_tests/tests/test-postgres-docker.rec:11]: Fixed

**Pull request merged**